meson.build

```diff
@@ -0,0 +1,10 @@
+project('eos-meta')
```

Comment:
I eyeballed it and decided it was fine, but you inspired me to check more closely and there are some slight differences between the `ninja dist` tarball and the one in OBS:

* Meson's `ninja dist` target generates a `.tar.xz`, whereas our fallback uses `.tar.gz`
* Files in Meson's `.tar.xz` have prefix `eos-meta/` whereas our fallback uses (eg) `eos-meta-1.5.0+dev1483.6470161/`
* I unpacked the two tarballs and ran `org.diffoscope.Diffoscope --exclude-directory-metadata --no-default-limits --max-page-size 40960000  --html eos-meta.html old/eos-meta-1.5.0+dev1483.6470161 new/eos-meta` on them. [Here's the output](https://willthompson.co.uk/misc/eos-meta.html). Interesting stuff:
    * `debian/source/options` is `--compression=gzip` in the tarball on OBS, and missing in Meson's; `debian/source/format` is `1.0` in the OBS tarball and `3.0 (native)` in the `meson dist` tarball.
    * All other changes are expected.

That said, I think all of the unexpected changes are actually things that `ci-integration/chroot-templates/obs-package` adjusts anyway, so this is all a red herring and practically speaking it does do the same thing.

Reply:
I'm just running `jenkins-obs` locally. Once it's finished and OBS has build the results from my home project, I'll diff the `.deb`s.

Reply:
Yeah, I'm pretty sure it will be fine. The `obs-package` helper hacks the source package in the expected way regardless of what's in the original tarball.